lib/magma/actions/update_attribute.rb

```diff
@@ -0,0 +1,48 @@
+require 'magma/actions/action_error'
+
+class Magma
+  class UpdateAttribute
+    def initialize(project_name, action_params = {})
```

Comment:
Could this be like `Magma::UpdateAttributeAction` or `Magma::Action::UpdateAttribute`?

Reply:
We renamed it to `Magma::UpdateAttributeAction` in https://github.com/mountetna/magma/pull/168/commits/05840ff7378d0818f27eb2ca9ce31623748549ee. It follows the same pattern we use for attributes - they live in `lib/magma/attributes` and are named like `Magma::BooleanAttribute`.